src/platform/site-wide/va-footer/helpers.jsx

```diff
@@ -10,13 +10,15 @@ export const FOOTER_COLUMNS = {
   RESOURCES: '2',
   CONNECT: '3',
   CONTACT: '4',
+  SUPERLINKS: 'bottom_rail',
```

Comment:
Lately, the links along the bottom are for some reason referred to `superlinks`, and the whole component the `superfooter` 🤔